test/MeshKernelNETTest/Api/MeshKernelTest.cs

```diff
@@ -3011,13 +3011,14 @@ public void Mesh2dUndoThenRedoTwoDeleteNodesThroughApi()
 
                     // Un-do
                     bool undone = false;
-                    Assert.AreEqual(0, api.UndoState(ref undone));
+                    int meshKernelId = -1;
+                    Assert.AreEqual(0, api.UndoState(ref undone, ref meshKernelId));
```

Comment:
add an assert that the ref argument has changed meshKernelId from its initial value of -1.